Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,77 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.tophead {
+  color: rgb(192, 208, 223);
```

Comment:
You didnt change the picture to make it dimmer, consider using 

 background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url('landingpagepic.jpg');
 background-position:center top;

Replace landingpagepic to the path to your image. The background-position means that the image will be centered.